Sunrise edits (#622)
graphprotocol/docs: website/pages/en/sunrise.mdx

## What is the Sunrise of Decentralized Data?

The Sunrise of Decentralized Data is an initiative spearheaded by Edge & Node. The goal is to enable subgraph developers to seamlessly upgrade to The Graph’s decentralized network.

This plan draws on many previous developments from The Graph ecosystem, including an upgrade Indexer to serve queries on newly published subgraphs, and the ability to integrate new blockchain networks to The Graph.

### What are the phases of the Sunrise of Decentralized Data?

**Sunray**: Enable support for hosted service chains, offer a seamless upgrade flow, offer a free plan on The Graph Network.  
**Sunbeam**: The upgrade window that subgraph developers will have to upgrade their subgraphs to The Graph Network. The start and length of this will be announced soon.  
**Sunrise**: Hosted service endpoints will expire as traffic moves to The Graph Network.

### How do I upgrade my hosted service subgraph?

> > Note: subgraphs upgraded to The Graph Network will not be available to revert back to the hosted service.

Reviewer: Just a wording suggestion:
```suggestion
> Note: Upgrading a subgraph to The Graph Network cannot be undone.
```

<VideoEmbed youtube="MLqJdVCom5k" />

To upgrade a hosted service subgraph, you can visit the subgraph dashboard on the [hosted service](https://thegraph.com/hosted-service).

1. Select the subgraph(s) you want to upgrade.
2. Select the receiving wallet (the wallet that will become the owner of the subgraph).
3. Click the "Upgrade" button.

Once your subgraph is published, the [upgrade Indexer](#what-is-an-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately!

### What happens if I don't upgrade my subgraph?

> Hosted service endpoints will expire at the end of the upgrade window.

Reviewer: Would be nice to have more details about when that is?

Author: CC: @azf20 are we clear to include a date for the upgrade window in tomorrow's docs release, or should we hold off on that? That date is not mentioned in the [blog](https://docs.google.com/document/d/1TIX0dHtoWDfIJsu9EArtYU-Rw5s6S0Jb6OBmj9hCk1s/edit) or social media copy.

Reviewer 2: This will be announced at a later date
